RecoEgamma/EgammaElectronProducers/python/lowPtGsfElectrons_cff.py

```diff
@@ -68,3 +68,61 @@
 from Configuration.Eras.Modifier_pp_on_AA_2018_cff import pp_on_AA_2018
 pp_on_AA_2018.toModify(lowPtGsfElectrons.preselection, minSCEtBarrel = 15.0)
 pp_on_AA_2018.toModify(lowPtGsfElectrons.preselection, minSCEtEndcaps = 15.0)
```

Comment:
I'd like to propose a less circular implementation with more similarity to what is done in the master version:
- define lowPtGsfElectronsPreRegression_cfi.py which would contain all of the above except for the module name changed to `lowPtGsfElectronsPreRegression`
- keep the rest of this file content in lowPtGsfElectrons_cfi.py
    - start with `lowPtGsfElectrons = lowPtGsfElectronsPreRegression.clone()`

Reply:
Done.

I also updated [these](https://github.com/cms-sw/cmssw/pull/33589/files#diff-aa405716300602f97f4c6e720176202fc9ab00be796fcd7a74504eff0bc3918fR66-R70) lines to read as 'lowPtGsfElectronsPreRegression.preselection' - I hope this is correct.